### ISSUE TITLE
using retinanet implementation for localization only, i.e num_classes = 0

### ISSUE THREAD

**Issue by rajaswa** [yhenon/pytorch-retinanet#61]
How can we proceed further with the current implementation to use it for a localization task only. I have a custom dataset with no classification requirements as such. I just need to detect the objects position and get the bounding boxes, object class label is not required

**Comment by pyaf** [comment]
Lolz

**Comment by astronaut71** [comment]
hi. Im facing same issue. Need to get the object position . Did you solve it?

**Comment by nathasha-naranpanawa** [comment]
Hi, I'm working on a similar issue. Did you get this working?

**Comment by astronaut71** [comment]
no. Is it possible to get the orientation of the detected objects with this?
